### PR TITLE
Feature/assest

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,600;1,300&family=Roboto:wght@500;700&display=swap"
```
Comment (blacksrc) [inline index.html]: You have added both `Open Sans` and `Roboto` here. Please, remove `Roboto`. The easy way is to create the link from google fonts again. The final url should have `Open+Sans` as `family`.

File: assets/css/general.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
```
Comment (blacksrc) [inline assets/css/general.css:2]: Fix this as well. Check this comment https://github.com/hedita/Acme-web-Solutions/pull/1/files#r785486933

File: index.html (hunk 1 above)
Comment (hedita) [reply index.html]: Done.

File: assets/css/general.css (hunk 2 above)
Comment (hedita) [reply assets/css/general.css:2]: Fixed.

Comment (blacksrc) [reply assets/css/general.css:2]: I cannot see any commit regarding to fixing this. Just to clarify this, you need to have the related font family only here. You are not using `Roboto`, etc. I added all `Open Sans` here but look at the `font-family`.
image

**Review by blacksrc — CHANGES_REQUESTED** [review]

File: assets/css/general.css (hunk 2 above)
Comment (blacksrc) [reply assets/css/general.css:2]: I cannot see any commit regarding this. Just to clarify this, you need to have the related font family only here. You are not using `Roboto`, etc. I added all `Open Sans` here but look at the `font-family`.

image

**Pull request merged**